Orange/preprocess/score.py

```diff
@@ -58,6 +58,14 @@ def __call__(self, data, feature=None):
 
         return self.score_data(data, feature)
 
+    def __repr__(self):
+        args = self.__class__.__init__.__code__.co_varnames
+        return "{}({})".format(
+            self.__class__.__name__,
+            ", ".join("{}={}".format(arg, repr(getattr(self, arg))) for i, arg in enumerate(args) if
+                arg != "self" and self.__class__.__init__.__defaults__[i-1] != getattr(self, arg))
+        )
```

Comment:
This is, again, something `Reprable` would unify in a single place. Note, `arg != 'self'` check can be avoided with:
```py
args = type(self).__init__.__code__.co_varnames[1:]
```